src/components/Startpage/Startpage.tsx

```diff
@@ -281,21 +281,21 @@ function Startpage() {
                 <Card style={{ width: "100%" }}>
                     <Card.Header>
                         <Card.Title><AnnouncementIcon /><span style={{ color: "#40ff00" }}> News / Announcements</span></Card.Title>
-                        <Card.Subtitle>Flipper item blacklist</Card.Subtitle>
+                        <Card.Subtitle>Minecraft Mod</Card.Subtitle>
                     </Card.Header>
                     <Card.Body>
-                        <p>The flipper now allows you to <b>blacklist</b> items you don't want to see again.</p>
-                        <p>The customization feature allows you to hide information in flips you don't want to see.</p>
+                        <p>We created a Minecraft mod as a QoL-Feature. It connects to the website and shows the flips you get there in your Minecraft chat for you to click.</p>
+                        <p>The mod can be downloaded on our <a target="_blank" rel="noreferrer" href="https://discord.gg/wvKXfTgCfb">discord</a> in the channel "mod-releases".</p>
```

Comment:
```suggestion
                        <p>The mod can be downloaded on our <a target="_blank" rel="noreferrer" href="https://discord.gg/wNRgeCYmW9">discord</a> in the channel "mod-releases".</p>
```
Replaced invite link with one that navigates to the mod-releases channel directly